tests/cli/validation/invalid-fulltext-directive/schema.graphql

```diff
@@ -0,0 +1,25 @@
+type _Schema_
+  @fulltext(
+    name: "jf"
+    language: EN
+    algorithm: RANKED
+    include: [
+      {
+        entity: "Musician",
+        fields: [
+          { name: 9 },
+          { name: "bio" },
+        ]
+      }
+    ]
+)
```

Comment:
Would be cool if this schema had a bunch of `@fulltext` directives that test all the different validations.

Reply:
Great idea! I've added to the test schema, so it triggers all 17 `@fulltext` validations errors.